README.md

```diff
@@ -11,13 +11,19 @@
 </p>
 
 <p align="center">
+  <img alt="Beta" src="https://img.shields.io/badge/status-beta-informational?style=for-the-badge">
+  <a href="https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.2.0">
+    <img alt="OpenTelemetry Java Instrumentation Version" src="https://img.shields.io/badge/otel-1.2.0-blueviolet?style=for-the-badge">
+  </a>
+  <a href="https://github.com/signalfx/splunk-otel-java-instrumentation/releases">
+    <img alt="GitHub release (latest SemVer)" src="https://img.shields.io/github/v/release/signalfx/splunk-otel-java?include_prereleases&style=for-the-badge">
+  </a>
+  <a href="https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-api">
+    <img alt="Maven Central" src="https://img.shields.io/maven-central/v/io.opentelemetry/opentelemetry-api?style=for-the-badge">
+  </a>
```

Comment:
Do we really need that? I find it confusing that we link to opentelemetry-api in the badge (which is one of the first things that you see when you open the page) when manual instrumentation is a more advanced and not that common use case.